server/controllers/post-form-controller.js

```diff
@@ -0,0 +1,17 @@
+const main = (req, res) => {
+	console.log('req.body.name:', req.body.name);
```

Comment:
I can't tell from the diff, but did you use the express body-parser or similar to get access to the `req.body` object?

Reply:
I had used body-parser. Hm I wonder why it's not showing up... I'll re-add it.